test/run_test.py

```diff
@@ -91,6 +92,8 @@
     'distributed/rpc/test_process_group_agent',
     'distributed/rpc/test_tensorpipe_agent',
     'distributed/test_distributed',
+    'distributed/test_distributed_spawn',
+    'distributed/test_ddp_under_dist_autograd',
```

Comment:
Why ddp_under_dist_autograd?

Reply:
Looks like I incorrectly resolved some merge conflicts (here and below this file), this was deleted as part of the RPC test refactor in https://github.com/pytorch/pytorch/pull/40818. Will update to correct this.